src/solutions/dae_solutions.jl

```diff
@@ -39,6 +39,7 @@ struct DAESolution{T, N, uType, duType, uType2, DType, tType, P, A, ID, DE} <:
     interp::ID
     dense::Bool
     tslocation::Int
+    sym_map::MType
```

Comment:
Can this be done using `prob.f.sys`? Requiring explicit maps isn't great.

Reply:
no, because save_idxs isn't known at sys construction stage, only at solve

Reply:
checking out the OrdinaryDiffEq part of this makes this all make a lot more sense to me.